frontend/test/__support__/cypress.js

```diff
@@ -222,6 +222,10 @@ export const describeWithToken = Cypress.env("HAS_ENTERPRISE_TOKEN")
   ? describe
   : describe.skip;
 
+export const itOpenSourceOnly = Cypress.env("HAS_ENTERPRISE_TOKEN")
+  ? it.skip
+  : it;
+
```

Comment:
@nemanjaglumac thoughts on this? It's the opposite of `describeWithToken` but for `it`. I figured that EE-behavior is typically tested in multiple test blocks, weird OSS-only behavior like this is probably a single test. I still think the it-vs-describe difference is a bit weird though.

Reply:
I am ok with it. It's just the name that feels so weird. I'm expecting that it "opens" the source of the code that follows :D
Having said that, I can't think of a nice name.

Reply:
Yeah agreed the name is weird. I don't expect it to be commonly used, but maybe a better name will come to us whenever it's next needed.